credentials.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"758252615955-m0jitbvb7ac8gi0ass2i9mnmoosejof9.apps.googleusercontent.com","project_id":"test-project-1533752905158","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://www.googleapis.com/oauth2/v3/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"PX1PyxSjEdeM3EYdD47U6OOm","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
errr let's not commit this to the repo... at the very least let's put it in a private google doc, and put in the README that you need to go get this file and put it in this location so it can do the auth goodness